components/service/glean/build.gradle

```diff
@@ -15,7 +15,7 @@ apply plugin: 'kotlin-android'
  * This uses a specific version of the schema identified by a git commit hash.
  */
 String GLEAN_PING_SCHEMA_GIT_HASH = "1933f26"
```

Comment:
Is this unused now and should be removed?

Reply:
No, this is a transient thing: once schema changes are merged I'll need to fix this up and revert the line below. See my comment [here](https://github.com/mozilla-mobile/android-components/pull/2516#discussion_r269037330). We need to do this otherwise tests will fail in this repo until schema changes are merged.